### DISCUSSION TITLE
I am suggesting: Add subscriptions boxes to a list be blocked?

### DISCUSSION THREAD

**Discussion by h2559** [uBlockOrigin/uBlock-issues#2504] [Ideas] [answered]
See title, some are bad, they take up 50% of the screen. the last example screenshot is really bad, 2 on screen subscription boxes + a side popup (and the ad, but that will be blocked as soon as i turn back on ublock origin), it is full screen coverage, it is not a popup, you can actually scroll through that on the main page of entreprenuer.com if you want to check it out.

these subscription boxes are annoying, they are a type of ad, they are visual clutter and spam, i keep having to manually block them, if there was a list to load which ones to block, then it would be easier. 

and in a way these subscriptions are also unwanted notifications, on my phone i always turn off most apps notifications, otherwise im hit with so much information overload due to the notifications pushed to me. 

i think having a new list for subscriptions to be blocked would be a good idea. it can be an optional feature to be enabled at first, just like the lists in the Filter Lists tab, and we can see how it goes from there? if it is popular then it can be enabled by default, otherwise just leave it as an option for those who care? ublock origin is an _everything_ blocker, lets get some ideas in if this can be done.

<img width="1383" alt="1" src="https://user-images.githubusercontent.com/20052727/220158201-5305c594-6f47-4d1c-babb-e18d2044706a.png">

<img width="1370" alt="2" src="https://user-images.githubusercontent.com/20052727/220158203-932d2bb1-acd5-425d-a983-79332d61a187.png">

<img width="1384" alt="3" src="https://user-images.githubusercontent.com/20052727/220158204-c03b08fc-ea56-4fb2-844b-73fd8f130624.png">

**Comment by u-RraaLL** [comment, marked as answer]
It's called "Annoyances".

![image](https://user-images.githubusercontent.com/92610420/220161266-65d20018-67e0-4eb9-8033-2a9774ba8a08.png)

**Reply by h2559** [reply]
ok, which one has specifically has subscriptions, i clicked on the Eye icon and the house icon for all those lists, most block social media and cookie popups, etc, but i did not read anything that says subscription boxes, i might have missed it.

**Reply by garry-ut99** [reply]
I guess The "Annoyances" ones: AdGuard Annoyances and Fanboy's Annoyance, just enable them and browse web to see.

**Reply by h2559** [reply]
It doesn't work, but it is ok, I don't care for this feature anymore, it is a huge job, and blocking ads is already a huge job too, adding more to uBlock Origin will be too much.

**Reply by stephenhawk8054** [reply]
If it doesn't work, report here:

https://github.com/easylist/easylist/issues

https://github.com/AdguardTeam/AdguardFilters/issues

**Reply by gorhill** [reply]
The first one is taken care by _"Fanboy – Annoyances"_ (or _"EasyList – Newsletter Notices"_ in next release), and I verified it works. The second one needs to be reported, `entrepreneur.com###hsNewletterMidHomepage-container` works.

**Reply by Yuki2718** [reply]
Fixed both in AdGuard Popup filters (or AdGuard – Popup Overlays in the next uBO release) included in AdGuard Annoyances, except that I coudln't reproduce the bottom-left popup in the second screenshot. Please provide URL links and don't let us to look screenshots hard. A very common illusion people tend to have is that there would be some magical filter lists which already satisfy all his needs. The reality is filter lists improves gradually by user's reports.

**Reply by h2559** [reply]
I understand now, but HOW do I report stuff like this? I have been using uBO for years and don't know how to do such a thing, or even know it exists. (And before uBO, I used the Adblock extension and they never had a way to report ads, so both extensions are bad in such ways). 

Imagine if you found a way that is OBVIOUS and EASY to report a problem or ad, you would literally be flooded with reports and then you can improve uBO so much! 

uBO really dropped the ball on this one, I would imagine less than 1 in 1,000,000 uBO users send in reports because there is no obvious way to do it.

**Reply by stephenhawk8054** [reply]
Go to the links here, click on "New issue" and follow instructions: https://github.com/uBlockOrigin/uBlock-issues/discussions/2504#discussioncomment-5995767

**Reply by h2559** [reply]
I see, I am going to write the obvious: You really can't expect 99.99999% of all people to sign up for a github account, learn to navigate some new unknown website (to them) to report an ad. 

I thought there was a way to do a report in the settings of the uBO extension. But if that is not possible, I will bookmark the links you gave for future reference, only because I already have a github account. 

TBH if I didn't have a github account I wouldn't register for a new one just to report an ad, you just lost 99.99999% of your potential ad reports, as there are many people out there like that.

**Reply by stephenhawk8054** [reply]
uBO already has a report button for those who don't know where to report yet: https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools

Since you are already having a github account and already make an issue/discussion, that's why I reference the links there.

**Reply by iam-py-test** [reply]
> You really can't expect 99.99999% of all people to sign up for a github account, learn to navigate some new unknown website (to them) to report an ad.

Then how else could people report issues?
uBo could have a report form like AdGuard, but that requires a server which gorhill controls, which costs money, takes time to setup and maintain, and goes against uBo's policy of not having a home server.
You _could_ report issues on Reddit, but the Reddit interface is horrible.
Your argument could be applied to Reddit too. A lot of people don't have Reddit accounts either.

**Reply by gorhill** [reply]
> you just lost 99.99999% of your potential ad reports

There are 10s of 1000s of filters in filter lists, which means that yes, a lot of people are willing to report for the benefit of all, and for your benefit. So it looks like the real obstacle is just a mindset one.

**Reply by Yuki2718** [reply]
> you would literally be flooded with reports

No problem if the number of contributor increases accordingly. From the very beginning uBO has always been a project of user, by user, and for user. All people working here are also just user like you. I know many people misunderstand as if a profit project. 

> You really can't expect 99.99999% of all people to sign up for a github account,

In that case the problem will be left and all user who consider this as a problem will suffer too, just that. It's a matter whether one takes some time to improve his own and potentially others' experience or not, IOW whether has volunteering mindset - a real case of Jewish tale of The big barrel of wine. Annonymous report system requires us to host server which we won't do, uBO doesn't send anything to remote server and server costs money. We also accept reports on Reddit where no strict template can be enforced, but then lots of reports there are not reproducible and more of noises wasting volunteers' time.

**Reply by h2559** [reply]
> > You really can't expect 99.99999% of all people to sign up for a github account, learn to navigate some new unknown website (to them) to report an ad.
> 
> Then how else could people report issues? uBo could have a report form like AdGuard, but that requires a server which gorhill controls, which costs money, takes time to setup and maintain, and goes against uBo's policy of not having a home server. You _could_ report issues on Reddit, but the Reddit interface is horrible. Your argument could be applied to Reddit too. A lot of people don't have Reddit accounts either.

Ah, I didn't know that there was a uBO policy of not having a home server. 

Maybe that info could be posted somewhere visible, so when people look to submit an ad report they would know why the things are the way they are for uBO, then users would be more forgiving.

**Reply by gorhill** [reply]
[Privacy policy:](https://github.com/gorhill/uBlock/wiki/Privacy-policy)

> uBO has no home server.

I think it's easy enough to find, that privacy policy is linked in many places, including the web stores from where to install uBO.

> users would be more forgiving

I think they are (you are ascribing your mindset to other people), and as for the current suggestion, I think it's all fine, we are not really receiving negative feedback with the current way of reporting.

**Reply by h2559** [reply]
I see, after reading this thread I think you all have it under control, I decided I don't even need to report any ads from now on because there are already enough people reporting ads, I will just continue to use this extension as a user, it is one of the best ones for Chrome.

**Reply by Yuki2718** [reply]
> I decided I don't even need to report any ads from now on because there are already enough people reporting ads

I don't think what you have raised here (`cityam.com` and `entrepreneur.com`) would have been addressed if you didn't. That sounds rather the big barrel of wine mindset, i.e. "others will do" which is quite discouraging aspect of involving in this project. At least then don't complain anything, as uBO has always been [For users by users](https://github.com/gorhill/uBlock#about).
